stacks-signer/src/config.rs

```diff
@@ -21,24 +21,22 @@ use std::path::PathBuf;
 use std::time::Duration;
 
 use blockstack_lib::chainstate::stacks::TransactionVersion;
-use blockstack_lib::util_lib::boot::boot_code_id;
-use clarity::vm::types::QualifiedContractIdentifier;
-use hashbrown::HashMap;
+use hashbrown::{HashMap, HashSet};
 use serde::Deserialize;
 use stacks_common::address::{
     AddressHashMode, C32_ADDRESS_VERSION_MAINNET_SINGLESIG, C32_ADDRESS_VERSION_TESTNET_SINGLESIG,
 };
 use stacks_common::consts::{CHAIN_ID_MAINNET, CHAIN_ID_TESTNET};
 use stacks_common::types::chainstate::{StacksAddress, StacksPrivateKey, StacksPublicKey};
 use stacks_common::types::PrivateKey;
-use wsts::curve::ecdsa;
+use wsts::curve::point::Point;
 use wsts::curve::scalar::Scalar;
 use wsts::state_machine::PublicKeys;
 
-/// List of key_ids for each signer_id
-pub type SignerKeyIds = HashMap<u32, Vec<u32>>;
-
 const EVENT_TIMEOUT_MS: u64 = 5000;
+// Default transaction fee in microstacks (if unspecificed in the config file)
+// TODO: Use the fee estimation endpoint to get the default fee.
+const TX_FEE_MS: u64 = 10_000;
```

Comment:
This name could be a little confusing. It might be more obvious if it was `TX_FEE_USTX`. Elsewhere in the code uses `ms` for milliseconds and `ustx` for microstacks.

Reply:
Done.